docs/markdown/ios/toast.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Toast
+description: Toasts are brief and small messages that overlay content, but do not block the user’s flow, as they are out of the way and ephemeral. Toasts do not require user action and primarily acknowledge that a user has performed an action or completed a task.
+fullwidth: true
+---
+
+<ImgContainer src="https://i.pinimg.com/originals/27/32/0d/27320d8b7731c7cea23fbf917e7d2160.jpg" noPadding alt="an example of toast"/>
+
+## Usage guidelines
+
+<TwoCol>
+  <Group>
+    <Do title="When to use" />    
+    - Briefly acknowledging a user action without interrupting their flow
+    - When acknowledging an action that relates to another surface, providing a link that navigates the user to that surface
+    - To undo actions after acknowledgement, if there isn’t already a way to do so on the current surface
+    - For system processes like showing that a process is loading, or when there are internet connectivity issues.
+  </Group>
+  <Group>
+  <Dont title="When not to use" />
+  - When, due to an error, a user can’t even continue performing basic tasks like browsing already loaded Pins.
+  - When asking a user to confirm that they want to perform an action. Use [Sheet](/ios/sheet) instead.
+  - When you want to suggest a user spend more money or try new features; use [Upsell](/upsell) instead.
+  - For errors that relate to a specific section or page. Use [Callout](/web/callout) or [SlimBanner](/slimbanner) instead.
+  - To guide or educate the user. Use [Popover](/popover) or [Tooltip](/tooltip) instead.
```

Comment:
make sure these all have /web in front of them

Reply:
Done